src/lib/sql/sql_translator.cpp

```diff
@@ -266,6 +277,26 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_select_statement(cons
   return _current_lqp;
 }
 
+void SQLTranslator::_translate_hsql_with_description(hsql::WithDescription& desc) {
+  SQLTranslator with_translator{_use_mvcc, nullptr, _parameter_id_allocator, _with_descriptions};
+  const auto lqp = with_translator._translate_select_statement(*desc.select);
+
+  // Save mappings: ColumnID -> ColumnName
+  std::unordered_map<ColumnID, std::string> column_names;
+  for (auto column_id = ColumnID{0}; column_id < lqp->column_expressions().size(); ++column_id) {
+    const auto identifiers =
+        with_translator._sql_identifier_resolver->get_expression_identifiers(lqp->column_expressions()[column_id]);
```

Comment:
How do you obtain identifiers from `_inflated_select_list_expressions`?
```
std::vector<std::shared_ptr<AbstractExpression>> _inflated_select_list_expressions
``` 

Unfortunately, I don't get your idea.

Reply:
Good point. 
For this change I have to merge the current master though. Before, we should merge #1753 to avoid migration efforts regarding `LQPView`.

Reply:
I have to trust you that you will address this because I will not be available until Aug. 6. ;)